.pre-commit-config.yaml

```diff
@@ -22,6 +22,12 @@ repos:
       - id: trailing-whitespace
         args: [--markdown-linebreak-ext=md]
 
+  # pull mirror of https://github.com/fsfe/reuse-tool
+  - repo: https://gitlab.ba.rivosinc.com/pub/it/reuse-tool
+    rev: v1.1.2
+    hooks:
+      - id: reuse
```

Comment:
This is still behind VPN, right? I guess the above one is too, so this was already broken for external users.

Reply:
Yes.  Probably makes sense to move all the URLs to the publically availble ones.   I'll change that.